From 8f26c30ca30484853c8771d1e9c04e38c4c0d79a Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 7 May 2018 19:25:48 +0800
Subject: [PATCH] soc: rockchip: system-status: include
 rockchip-system-status.h

Fix the following warning(s):
drivers/soc/rockchip/rockchip-system-status.c:24:5: warning: no previous prototype for rockchip_register_system_status_notifier [-Wmissing-prototypes]
drivers/soc/rockchip/rockchip-system-status.c:31:5: warning: no previous prototype for rockchip_unregister_system_status_notifier [-Wmissing-prototypes]
drivers/soc/rockchip/rockchip-system-status.c:46:6: warning: no previous prototype for rockchip_set_system_status [-Wmissing-prototypes]
drivers/soc/rockchip/rockchip-system-status.c:70:6: warning: no previous prototype for rockchip_clear_system_status [-Wmissing-prototypes]
drivers/soc/rockchip/rockchip-system-status.c:98:15: warning: no previous prototype for rockchip_get_system_status [-Wmissing-prototypes]

Change-Id: Ic3fb19a4a23f8f6c6833a7ebc573617c372d51bd
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/soc/rockchip/rockchip-system-status.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/rockchip/rockchip-system-status.c b/drivers/soc/rockchip/rockchip-system-status.c
index 56d5ae3be08c..5012b8818a53 100644
--- a/drivers/soc/rockchip/rockchip-system-status.c
+++ b/drivers/soc/rockchip/rockchip-system-status.c
@@ -13,6 +13,7 @@
 
 #include <linux/mutex.h>
 #include <linux/notifier.h>
+#include <soc/rockchip/rockchip-system-status.h>
 
 static unsigned long system_status;
 static unsigned long ref_count[32] = {0};
-- 
2.35.3

